.github/workflows/release.yml

```diff
@@ -133,15 +137,15 @@ jobs:
         run: |
           export PATH=$CARGO_HOME/bin:$PATH
           # Actually do builds and make zips and whatnot
-          cargo dist build --tag=${{ github.ref_name }} --output-format=json ${{ matrix.dist-args }} > dist-manifest.json
+          cargo dist build --tag="${GITHUB_REF_NAME}" --output-format=json "${MATRIX_DIST_ARGS}" > dist-manifest.json
```

Comment:
I believe we want `MATRIX_DIST_ARGS` to remain exploded. This will have the shell treat `MATRIX_DIST_ARGS` as a single string containing whitespace, but the data inside is actually several distinct argument strings.